src/components/Thresholds/ThresholdFormSection.tsx

```diff
@@ -0,0 +1,184 @@
+
+import { config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { ThresholdValues } from 'contexts/SuccessRateContext';
+
+interface LabelProps {
+  color: string;
+  title: string;
+}
+
+interface ThresholdSectionProps {
+  label: string;
+  unit: '%' | 'ms';
+  description: string;
+  thresholds: ThresholdValues;
+  setThresholds: (threshold: ThresholdValues) => void;
+}
+
+const getDotStyles = (color: string) => (theme: GrafanaTheme2) => ({
+  container: css`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    margin-bottom: 5px;
+    margin-top: 5px;
+    width: 80px;
+  `,
+  dot: css`
+    background-color: ${color};
+    width: 15px;
+    height: 15px;
+    border-radius: 50%;
+    margin-right: 10px;
+  `,
+});
+
+const getSectionStyles = () => (theme: GrafanaTheme2) => ({
+  estimate: css`
+    margin-bottom: 0px;
+    font-style: italic;
+    align-self: center;
+    font-size: 0.7rem;
+  `,
+  disabled: css`
+    // This is a hack, can't seem to get the disabled prop on Input to work
+    div > input {
+      cursor: not-allowed;
+      background-color: rgba(204, 204, 220, 0.07);
+      color: rgba(204, 204, 220, 0.4);
+      border: 1px solid rgba(204, 204, 220, 0.07);
+    }
+  `,
+  group: css`
+    margin-top: 25px;
+    margin-bottom: 25px;
+  `,
+});
+
+const Dot = ({ color, title }: LabelProps) => {
+  const styles = useStyles2(getDotStyles(color));
+  return (
+    <div className={styles.container}>
+      <div className={styles.dot}></div>
+      <strong>{title}</strong>
+    </div>
+  );
+};
+
+const displayDowntimeEstimate = (percentage: number) => {
+  // 43800 minutes per month
+  // 730 hours per month
+  const minutesOfUptime = (percentage / 100) * 43800;
+  const hoursOfDowntime = 730 - minutesOfUptime / 60;
+
+  if (hoursOfDowntime < 1) {
+    const minutes = hoursOfDowntime * 60;
+    return `This translates to approximately ${minutes.toFixed(0)} minutes of downtime per month or ${(
+      minutes / 31
+    ).toFixed(0)} minute(s) per day.`;
+  } else if (Number(hoursOfDowntime.toFixed(0)) === 1) {
+    return `This translates to approximately one hour of downtime per month`;
+  } else {
+    return `This translates to approximately ${hoursOfDowntime.toFixed(0)} hours of downtime per month.`;
+  }
+};
+
+const ThresholdFormSection = ({ label, unit, description, thresholds, setThresholds }: ThresholdSectionProps) => {
+  const styles = useStyles2(getSectionStyles());
+  const isLatency = unit === 'ms';
+  const handleUpdateThreshold = useCallback(
+    (key: string, newValue: number) => {
+      const newThresholds = {
+        ...thresholds,
+        [key]: newValue,
+      };
+      setThresholds(newThresholds);
+    },
+    [thresholds, setThresholds]
+  );
+
+  const After = <div style={{ display: 'flex', alignItems: 'center', padding: '5px' }}>{unit}</div>;
```

Comment:
[nit] consider adding these styles to your `getStyles` func and passing as a classname